docs/dashboards/detection-response-dashboard.asciidoc

```diff
@@ -0,0 +1,40 @@
+[[detection-response-dashboard]]
+= Detection & Response dashboard
+
+The Detection & Response dashboard provides focused visibility into the day-to-day operational activities of your security environment. Security operations managers and analysts can quickly monitor recent and high priority detection alerts and cases, and identify the hosts and users associated with frequent and severe alerts.
+
+[role="screenshot"]
+image::detections/images/detection-response-dashboard.png[Overview of Detection & Response dashboard]
+
+Interact with various dashboard elements:
+
+* Use the date and time picker in the upper-right to specify a timeframe for displaying information on the dashboard. 
+
+* In sections that list alert counts, click the number to investigate the alerts in Timeline.
+
+* Click the names of detection rules, cases, hosts, and users to open their details pages.
+
+The following sections are included:
+
+[width="100%",cols="s,"]
+|==============================================
+
+|Alerts
+|The total number of detection alerts generated in the timeframe, organized by status and severity. Select *View alerts* to open the Alerts page.
+
+|Cases
+|The total number of cases created in the timeframe, organized by status. Select *View cases* to open the Cases page.
+
+|Open alerts by rule
+|The top four detection rules with alerts in the *Open* status, organized by the severity and number of alerts for each rule. Select *View all open alerts* to open the Alerts page.
```

Comment:
```suggestion
|The top four detection rules with open alerts, organized by the severity and number of alerts for each rule. Select *View all open alerts* to open the Alerts page.
```
I could be off base with removing the bold on "open", but I think we usually use bold on things users click, and here I think open is just an adjective.